server/amp/handlers/story-page.js

```diff
@@ -86,7 +85,7 @@ async function ampStoryPageHandler(
     )
       return res.redirect(story.url);
 
-    const seoInstance = getSeoInstance(seo, config, "story-page-amp");
+    const seoInstance = typeof seo === "function" ? seo(config, "story-page-amp") : seo;
     const seoTags =
```

Comment:
`getSeoInstance` helper was not needed since it's a one liner and isn't being used anywhere else